tests/components/technove/test_binary_sensor.py

```diff
@@ -16,6 +16,12 @@
 from tests.common import MockConfigEntry, async_fire_time_changed
 
 
+@pytest.fixture
+def platforms() -> Platform | list[Platform]:
```

Comment:
Why is this added

Reply:
hmm, I can't remember why this would be added here and in sensor.py. I removed it.